documentation/guides/breakpoints-migration.md

```diff
@@ -0,0 +1,792 @@
+## New breakpoints
+
+TODO: write about the 5 new breakpoints
```

Comment:
Wanted to drop in some content we've used for comms during the project that could be helpful inspiration for this section! Definitely not something that we could just use "as is" for the section but it has some good info we could rip out. 

> As part of our project to reduce the total number of unique breakpoints in polaris and Shopify’s admin we've created the follow set of breakpoints: 
> 
> * `xs`: 0px
> * `sm`: 490px
> * `md`: 768px
> * `lg`: 1040px
> * `xl`: 1440px
> 
> A script takes these values and generates Sass variables (which can be used in media conditions) for the breakpoint and up, for the breakpoint and down, and for only the breakpoint.
> 
> Example of generated output for `breakpoints-md`:
> ```
> @media #{$p-breakpoints-md-up} {/*...*/}
> @media #{$p-breakpoints-md-down} {/*...*/}
> @media #{$p-breakpoints-md-only} {/*...*/}
> ```

Perhaps we want to also add something about having to use/import Polaris tokens to use these values?